tests/python/frontend/pytorch/test_forward.py

```diff
@@ -379,6 +379,27 @@ def test_forward_maxpool1d():
                                     stride=2).eval(),
                 input_data)
 
+def test_forward_split():
+    torch.set_grad_enabled(False)
+    input_shape = [4, 10]
+
+    class Split1(Module):
+        def forward(self, *args):
+            return torch.split(args[0], 2, 0)
+
+    class Split2(Module):
+        def forward(self, *args):
+            return torch.split(args[0], [2, 3, 5], 1)
```

Comment:
Does this end up testing `split_with_sizes`?

Reply:
Yes, the `Split2` will test `split_with_sizes`.